cmd/restart.go

```diff
@@ -38,11 +39,12 @@ func Restart() *cobra.Command {
 
 	cmd.Flags().StringVarP(&devPath, "file", "f", defaultManifest, "path to the manifest file")
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "namespace where the exec command is executed")
+	cmd.Flags().BoolVarP(&wait, "wait", "w", false, "wait until pods are ready")
```

Comment:
What about doing this the default (and maybe the unique) option?

Reply:
Im 50/50 on this one. Do you know if anyone other than us is using this command?